app-admin/pinkybar/metadata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+	</maintainer>
+	<maintainer type="person">
+		<email>sacewe@emailure.net</email>
+		<name>Aaron Caffrey</name>
+	</maintainer>
+	<upstream>
+		<remote-id type="github">su8/pinky-bar</remote-id>
+	</upstream>
+	<use>
+		<flag name="alsa">To get the sound volume level.</flag>
+		<flag name="colours">Colorize the output data</flag>
```

Comment:
Gentoo is using AmE, so 'colors', please.

Reply:
Renamed the variable.